index.html

```diff
@@ -0,0 +1,105 @@
+            <li><a href="#program">Join</a></li>
+            <li><a href="#contactform">Sponsor</a></li>
+            <li><a href="#contactform">News</a></li>
+          <li><a href="#contactform"></a><button type="button" class="campaign">CC Campaign</button></a></li>
+        </ul>
+    </nav>
+    <section class="home" id="home">
+        <section class="inner">
+        <h1>Environmental <br> Development</br>2023</br></span></h1>
+        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit.Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+        </p>
+        <span class="date"><h2>2023.10.25(THU)~16(FRI)</h2>
+                          <p>@Yezin Agricultrual University, Myanmar</p>
+        </span>
+    </section>
+</section>
+
+<section class="program" id="program"><h2>Main <u>Pro</u>gram</h2>
+    <article class="container1">
+        <card class="mycard">
+            <img src="images/lecture (1).png" alt="program">
+            <h3>Lecture</h3>
+        <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+            
+        </p> 
+    </card>
+        <card class="mycard"><img src="images/cc.png" alt="program"/>
+            <h3>CC Exhibition</h3>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                
+           </p>
+        </card>
+        
+        <card class="mycard"><img src="images/forum.png" alt="program"/>
+            <h3>Forum</h3>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                
+           </p></card>
+        <card class="mycard"><img src="images/workshop.png" alt="program"/>
+            <h3>Workshop</h3>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+               
+           </p></card>
+        <card class="mycard"><img src="images/ignite.png" alt="program"/>
+            <h3>CC Ignite</h3>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                
+           </p></card>
+    </article>
+    <h4>SEE  THE  WHOLE  PROGRAM</h4>
+    <button class="join">Join The Program 2023</button>
+</section>
+<section class="group"id='bio'></section>
+<section class="sponsor"><h2>P<u>artn</u>ers</h2>
+    <ul>
+        <li><img src="images/mozilla 1.png" alt=""></li>
+        <li><img src="images/googlekorea 1.png" alt=""></li>
+        <li><img src="images/naver 1.png" alt=""></li>
+        <li><img src="images/daumkakaologo 1.png 1.png" alt=""></li>
+        <li><img src="images/airbnb 1.png" alt=""></li>
+    </ul>
```

Comment:
Nice job adding a descriptive `alt` value to some of your images as requested by the previous reviewers😉. But you still haven't added value to all of them as requested by the previous reviewers, kindly ensure every image has an `alt` value😉. As this helps with accessibility, aids screen readers for visually impaired people, and can serve as a means of description where the links to the images are broken or there's a network delay😉.